app/controllers/glossary_controller.rb

```diff
@@ -0,0 +1,5 @@
+class GlossaryController < ApplicationController
+  before_action :authenticate_user!, except: [:index]
```

Comment:
+1 to this being its own controller 🙌🏾

I think this can extend `PublicController` instead which skips authentication.